cluster/images/etcd/Makefile

```diff
@@ -18,6 +18,7 @@
 # 	[TAG=3.0.4] [REGISTRY=gcr.io/google_containers] [ARCH=amd64] [BASEIMAGE=busybox] make (build|push)
 
 TAG?=3.0.4
+REGISTRY_TAG?=3.0.4
```

Comment:
It should default to TAG. Otherwise someone setting TAG from the command line is going to blow away a 3.0.4 container.